src/stactools/sentinel2/stac.py

```diff
@@ -174,8 +175,24 @@ def create_item(
         raise ValueError(
             f"Could not determine EPSG code for {granule_href}; which is required."
         )
-    centroid = shapely.geometry.shape(item.geometry).centroid
-    projection.centroid = {"lat": round(centroid.y, 5), "lon": round(centroid.x, 5)}
+
+    if item.geometry and item.geometry.get("type") == "MultiPolygon":
```

Comment:
If `antimeridian_strategy` is set to `NORMALIZE`, you can end up with a centroid longitude greater than 180 or less than -180. Is that OK? The whole `NORMALIZE` option should probably go away at some point, IMO.

Reply:
I think that is okay, since the centroid is representative of the geometry. I'm only using SPLIT and this doesn't change the normalize behavior, so I'm not going to delve into that.